### PR TITLE
Date fetch

### PR DESCRIPTION
**Pull request by biswasarpita**

### CODE REVIEW

File: Controller/customerPolicyControl.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const moment = require('moment');
+const tz = require('moment-timezone')
```
Comment (amitroy) [inline Controller/customerPolicyControl.js]: Remove this module

File: Controller/customerPolicyControl.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+const moment = require('moment');
```
Comment (amitroy) [inline Controller/customerPolicyControl.js]: Remove this module

File: Controller/customerPolicyControl.js [hunk 3]
```diff
@@ -42,6 +44,42 @@ module.exports.get_customerdetails_by_id = (req, res) => {
...
+    try {
+        const result = await PolicyDetails.findOne({ policyNumber: req.params.policyNumber });
+        res.send({
+            status: 'OK',
+            policies: result
+        });
+    }
+    catch (e) {
+        res.status(400).send({
+            status: 'ERROR',
```
Comment (amitroy) [inline Controller/customerPolicyControl.js]: error instead of ERROR

File: Controller/customerPolicyControl.js [hunk 4]
```diff
@@ -42,6 +44,42 @@ module.exports.get_customerdetails_by_id = (req, res) => {
...
+    try {
+        const result = await PolicyDetails.find({ createdAt: { $gte: new Date(minDate), $lte: new Date(maxDate) } });
+        res.send({
+            status: 'OK',
+            policies: result
+        });
+    }
+    catch (e) {
+        res.status(400).send({
+            status: 'ERROR',
```
Comment (amitroy) [inline Controller/customerPolicyControl.js]: Keep ERROR formats in sync

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "customer_policy_details",
-  "version": "1.0.1",
+  "version": "1.2.0",
```
Comment (amitroy) [inline package.json]: Minor change, change version to 1.0.2

File: package.json [hunk 6]
```diff
@@ -14,6 +14,8 @@
     "dotenv": "^6.0.0",
     "express": "^4.16.3",
     "jsonwebtoken": "^8.3.0",
+    "moment": "^2.22.2",
```
Comment (amitroy) [inline package.json]: uninstall

File: package.json [hunk 7]
```diff
@@ -14,6 +14,8 @@
...
+    "moment-timezone": "^0.5.21",
```
Comment (amitroy) [inline package.json]: uninstall

**Review by amitroy — CHANGES_REQUESTED** [review]

File: Controller/customerPolicyControl.js (hunk 1 above)
Comment (biswasarpita) [reply Controller/customerPolicyControl.js]: removed

File: Controller/customerPolicyControl.js (hunk 2 above)
Comment (biswasarpita) [reply Controller/customerPolicyControl.js]: Done

**Review by amitroy — APPROVED** [review]

File: Controller/customerPolicyControl.js (hunk 3 above)
Comment (biswasarpita) [reply Controller/customerPolicyControl.js]: Done

**Review by amitroy — APPROVED** [review]

**Pull request merged**